set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     BSADBUARVDUJVEVFVGDALHJVHUNVI $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Real subtraction
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c -R $.
+
+  $( Real number subtraction. $)
+  crsub $a class -R $.
+
+  $( Based on ~ conventions-comments , I add a "(Contributed by...)" and a
+     "(Revised by...)" tag every time the proofs are equivalent.  I also add a
+     "Based on..." sentence so that any other contributors are credited.  In
+     these theorems the only difference is the use of ` -R ` instead of ` - ` .
+     All theorems are named and constructed based on the equivalent complex
+     number subtraction theorems. $)
+
+  ${
+    $d x y z $.
+    $( Define subtraction between real numbers.  This operator saves a few
+       axioms over ~ df-sub in certain situations.  Theorem ~ rsubval shows its
+       value, ~ rsubadd relates it to addition, and ~ rsubcl proves its
+       closure.  Based on ~ df-sub .  (Contributed by NM, 26-Nov-1994.)
+       (Revised by Steven Nguyen, 7-Jan-2022.) $)
+    df-rsub $a |- -R = ( x e. RR , y e. RR |->
+                      ( iota_ z e. RR ( y + z ) = x ) ) $.
+  $}
+
+  ${
+    $d x y A $.  $d x y B $.  $d ph x y $.
+    rnegmodv.a $e |- ( ph -> A e. RR ) $.
+    rnegmodv.b $e |- ( ph -> B e. RR ) $.
+    rnegmodv.1 $e |- ( ph -> E. y e. RR ( B + y ) = A ) $.
+    $( There is at most one real negative, deductive form.  (Contributed by
```

Comment:
`E!` is "exactly one" ; `E*` is "at most one"

Reply:
yeah I originally tried `E*` but then `E. => E!` was easier. Would `rnegeudv` still be the label?

Reply:
yes rnegeudv

Reply:
The comments must be adapted, too. And the following theorem(s) are also affected.